src/WebJobs.Script.WebHost/Startup.cs

```diff
@@ -29,6 +29,8 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IApplicationLifetime applicationLifetime, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
+            MaxConnectionHelper.SetMaxConnectionsPerServer(loggerFactory.CreateLogger($"Host.{nameof(MaxConnectionHelper)}"));
```

Comment:
Would probably be good to put a comment here describing that this is temporary (and perhaps linking to an issue to track the work to address the problem)

Reply:
That's in the helper already: 

        // Temporary helper until https://github.com/Azure/azure-storage-net/issues/580 is fixed.
        public static void SetMaxConnectionsPerServer(ILogger logger)